DisplayBundle/DisplayBlock/DisplayBlockManager.php

```diff
@@ -148,6 +148,23 @@ public function isPublic(ReadBlockInterface $block)
     }
 
     /**
+     * @param ReadBlockInterface $block
+     *
+     * @return string|null
+     */
+    public function toString(ReadBlockInterface $block)
+    {
+        /** @var DisplayBlockInterface $strategy */
+        foreach ($this->strategies as $strategy) {
+            if ($strategy->support($block)) {
+                return $strategy->toString($block);
+            }
+        }
+
+        return null;
```

Comment:
@alavieille : as ``toString`` must return a string value, why not returning '' in the case where no strategy is found ? That way, the rendering is not impacted in this case.